packages/devtools_app/lib/src/shared/memory/class_name.dart

```diff
@@ -149,22 +157,21 @@ class HeapClassName {
     return ClassType.dependency;
   }
 
-  bool get isCreatedByGoogle => isPackageless || isDartOrFlutter;
+  late final bool isCreatedByGoogle = isPackageless || isDartOrFlutter;
 
   /// True, if the library does not belong to a package.
   ///
   /// I.e. if the library does not have prefix
   /// `dart:` or `package:`.
   /// Examples of such classes: Code, Function, Class, Field,
   /// number_symbols/NumberSymbols, vector_math_64/Matrix4.
-  bool get isPackageless =>
-      library.isEmpty ||
+  late final bool isPackageless = library.isEmpty ||
       (!library.startsWith(PackagePrefixes.dart) &&
           !library.startsWith(PackagePrefixes.genericDartPackage));
 
   /// True, if the package has prefix `dart:` or has perfix `package:` and is
   /// published by Dart or Flutter org.
-  bool get isDartOrFlutter {
+  late final isDartOrFlutter = () {
```

Comment:
Ditto

Reply:
resolved in https://github.com/flutter/devtools/pull/5134